src/askExpressMiddleware/askExpressMiddleware.ts

```diff
@@ -16,8 +16,8 @@ export function askExpressMiddleware(
     next: NextFunction
   ) {
     try {
-      const askScriptCode = request.body.code;
-      const queryResponse = await runUntyped(environment, parse(askScriptCode));
+      const askCode = request.body.code;
+      const queryResponse = await runUntyped(environment, parse(askCode));
```

Comment:
`parse` should come from the askscript package now, not askcode :)

Reply:
We'll support both AskCode and AskScript inputs (even without requiring user to provide an option) , but for now we need AskScript support for playground

Reply:
@mhagmajer 
Please have a look now.
Also please look at the new test case. Is this a valid test for askscript?